packages/react-devtools-extensions/src/main.js

```diff
@@ -472,8 +472,8 @@ function createPanelIfReactLoaded() {
       let needsToSyncElementSelection = false;
 
       chrome.devtools.panels.create(
-        IS_CHROME || IS_EDGE ? '⚛️ Components' : 'Components',
```

Comment:
Can we keep the IS_EDGE here? since Edge still has the default UI that need this emoji version title.